_includes/misc/hunspell-dictionaries-path.md

```diff
@@ -0,0 +1,18 @@
+The `ephox.spelling.hunspell-dictionaries-path` property is used to define the location of the Hunspell dictionaries. When the setting is not provided, Hunspell dictionaries are not supported.
```

Comment:
Just noticed this says both `property` and `setting`. Should this be the same term?

Reply:
```suggestion
The `ephox.spelling.hunspell-dictionaries-path` setting is used to define the location of the Hunspell dictionaries. When the setting is not provided, Hunspell dictionaries are not supported.
```